src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -38,14 +40,24 @@
     private File dir;
     /**
      * Constructor creates new driver.
+     * Create a sub directory under default temp directory to set downloadPath
+     * Print path to tempDirectory
      */
     public ChplDownloadSteps() {
         driver = Hooks.getDriver();
         if (StringUtils.isEmpty(url)) {
             url = "http://localhost:3000/";
         }
         if (StringUtils.isEmpty(downloadPath)) {
-           downloadPath = "\\target\\download-files";
+            String tempDirectory;
+            try {
+              tempDirectory = Files.createTempDirectory("download-files").toString();
+              System.out.println(tempDirectory);
+            } catch (final IOException e) {
+              // Temp dir creation failed - use default
+              tempDirectory = null;
```

Comment:
This doesn't "use default". If the directory creation fails it will use `null`, and then the directory will be `new File(null)`, which doesn't look right